modules/nw-multus-ipvlan-object.adoc

```diff
@@ -1,87 +1,59 @@
 //
-// * networking/multiple_networks/configuring-ipvlan.adoc
+// * networking/multiple_networks/configuring-additional-network.adoc
 
-[id="nw-multus-ipvlan-object_{context}"]
-= Configuration for ipvlan
+//37.1. IPVLAN overview
+// https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_networking/getting-started-with-ipvlan_configuring-and-managing-networking#ipvlan-overview_getting-started-with-ipvlan
 
-The configuration for an additional network attachment that uses the ipvlan
-Container Network Interface (CNI) plug-in is provided in two parts:
+[id="nw-multus-ipvlan-object_{context}"]
+= Configuration for an IPVLAN additional network
 
-* Cluster Network Operator (CNO) configuration
-* CNI plug-in configuration
+The following object describes the configuration parameters for the IPVLAN CNI
+plug-in:
 
-The CNO configuration specifies the name for the additional network attachment
-and the namespace to create the attachment in. The plug-in
-is configured by a JSON object specified by the `rawCNIConfig` parameter in
-the CNO configuration.
+.IPVLAN CNI plug-in JSON configuration object
+[cols=".^2,.^2,.^6",options="header"]
+|====
+|Field|Type|Description
 
-The following YAML describes the configuration parameters for the CNO:
+|`cniVersion`
+|`string`
+|The CNI specification version. The `0.3.1` value is required.
 
-.Cluster Network Operator YAML configuration
-[source,yaml]
-----
-name: <name> <1>
-namespace: <namespace> <2>
-rawCNIConfig: '{ <3>
-  ...
-}'
-type: Raw
-----
-<1> Specify a name for the additional network attachment that you are
-creating. The name must be unique within the specified `namespace`.
+|`name`
+|`string`
+|The value for the `name` parameter you provided previously for the CNO configuration.
 
-<2> Specify the namespace to create the network attachment in. If
-you do not specify a value, then the `default` namespace is used.
+|`type`
+|`string`
+|
 
-<3> Specify the CNI plug-in configuration in JSON format, which
-is based on the following template.
-
-The following object describes the configuration parameters for the ipvlan CNI
-plug-in:
-
-.ipvlan CNI plug-in JSON configuration object
-[source,json]
-----
-{
-  "cniVersion": "0.3.1",
-  "name": "<name>", <1>
-  "type": "ipvlan",
-  "mode": "<mode>", <2>
-  "master": "<master>", <3>
-  "mtu": <mtu>, <4>
-  "ipam": { <5>
-    ...
-  }
-}
-----
-<1> Specify the value for the `name` parameter you provided previously for
-the CNO configuration.
+|`mode`
+|`string`
+|The operating mode for the virtual network. The value must be `l2`, `l3`, or `l3s`. The default value is `l2`.
 
-<2> Specify the operating mode for the virtual network. The value must
-be `l2`, `l3`, or `l3s`. The default value is `l2`.
+|`master`
+|`string`
+|Specify the ethernet interface to associate with the network attachment. If a `master` is not specified, the interface for the default network route is used.
```

Comment:
Construction of these descriptions isn't parallel. Could rewrite to make them so.